### DISCUSSION TITLE
General guidelines for MDN translated content に日本語版を追加する

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#698] [General]
5/20 のミートアップの中間報告会で上がった話題です。

https://github.com/mdn/translated-content/tree/main/docs

translated content リポジトリーの docs フォルダには、各言語版毎のガイドラインがありますが、日本語版はまだありません。

翻訳ミートアップで使っている翻訳ガイドがあることもあり、今まで、定期的に頭の中に浮かびつつも、後回しにしていました。

これを機会に、日本語版を作ろうと思いますので、ご意見や内容を募集します。

**Comment by debiru** [comment]
この docs を書くに当たって質問があります。

- https://github.com/mdn/translated-content/tree/main/docs#translating-heading-ids

https://github.com/mdn/translated-content/issues/11562 この件ですが、なぜ日本語訳では上記ガイドラインに反して見出しの ID を翻訳してしまっているのでしょうか。そのようにした経緯や理由を教えてください。

**Reply by potappo** [reply]
経緯は以下の通りです。
https://github.com/mozilla-japan/translation/issues/586#issuecomment-1053739847
「Markdown化以降、IDは各国語に翻訳する」方針が最新で、[#translating-heading-ids](https://github.com/mdn/translated-content/tree/main/docs#translating-heading-ids)の記載は古いように思えます。

**Reply by debiru** [reply]
情報ありがとうございます。あら、そうだったのですね。[#translating-heading-ids](https://github.com/mdn/translated-content/tree/main/docs#translating-heading-ids) の記載が古いのは困りますね。英語版のその部分、修正できそうだったら Pull-Req を出しておこうと思います。

**Comment by debiru** [comment]
他の国の翻訳ガイドも読んだのですが、以下のような内容で日本語版の /docs/ja 翻訳ガイドを書いてみました。

https://github.com/mdn/translated-content/pull/13353

（軽微な修正も含めて）翻訳を初めてやってみたいけれど、何をしていいのか分からない人向けの文章にすることを意識して書きました。

とりあえず書いてみたという程度の下書きというか仮文書なので、修正・加筆が必要な可能性が高いです。もし気づいたことがあればコメントください。あまり内容が適切そうでなければ PR を却下していただいても結構です。

本質的な内容（翻訳作業におけるルール）は /docs/ja ではなく https://mozilla-japan.github.io/mdn-translation-guide/ に書いてしまったほうがいいかなと思ったので /docs/ja にはほとんど書きませんでした。

- PRのプレビューは https://github.com/debiru/translated-content/blob/docs_ja_guide/docs/ja/README.md です

**Reply by potappo** [reply]
大筋は良さそうに思いますが、細かいところは仕事後に再度見ます！

**Reply by potappo** [reply]
「This branch cannot be rebased due to conflicts」 と出ていて、最新のメインとのマージが必要です。PR ブランチを修正していただけば、メインにマージしますので、よろしくお願いします。

**Reply by debiru** [reply]
あれ、本当ですか。そのようには見えていませんでしたが、最新のメインとのマージをしてみました。これでマージできるでしょうか。ご確認をお願いいたします。

**Reply by potappo** [reply]
ありがとうございます。main ブランチにマージしました。結局、再度、マージしないといけなかったので、タイミングの問題ですね。たまたま、他言語で色々修正があってすんなりとマージできなかったようです。

**Comment by potappo** [comment]
日本語版が追加されましたが、しばらく、このディスカッションはオープンしたままにしておきますので、修正や追記の意見などがあれば、どなたでも、コメントをお願いします。
